### ISSUE TITLE
[CRASH] [1.17.1 U_G9] crash on start on (potentially old) intel IGPU driver linux (Ubuntu)

### ISSUE THREAD

**Issue by Chaos02** [sp614x/optifine#6007]
## Description of Issue
Minecraft doesn't start with optifine installed on Ubuntu (21.04) (Wayland) with an Intel IGPU from my 2012 Macbook Pro 9,2 13'' with error  message 
```
GLFW error 65543: GLX: Failed to create context: GLXBadFBConfig.

Please make sure you have up-to-date drivers (see aka.ms/mcdriver for instructions).
```
On Windows the version works just fine so the hardware supports it.
Linux graphics drivers (for intel) are supplied by the Distro maintainers and you cannot find any more-up-to-date packages for them either.
This is what `lspci -k | grep -EA3 'VGA|3D|Display'` prints for me:
```
00:02.0 VGA compatible controller: Intel Corporation 3rd Gen Core processor Graphics Controller (rev 09)
	Subsystem: Apple Inc. 3rd Gen Core processor Graphics Controller
	Kernel driver in use: i915
	Kernel modules: i915
```

## Steps to Reproduce
1. Open Minecraft launcher
2. select through installer installed Optifine 1.17.1 Profile (installed multiple times)
3. Hit Play and wait for something to happen
4. Error message.

## OptiFine Version
preview_OptiFine_1.17.1_HD_U_G9_pre26.jar

## Installation Method
Standalone installer

## Log Files/Crash Reports
[latest.log](https://pastebin.com/y7GAfCDV)
No Crash report generated.

## F3 Debug Screenshot
No Game Window.

## Prior Testing
Not happening in Vanilla, not happening in Windows (not VM).

**Comment by Ari-RERA-43** [comment]
Dupe of #5739

**Comment by Chaos02** [comment]
> Dupe of #5739 so you should close this issue. There is a fix for Linux in the comments of #5739.

Thank you very much indeed!
